File: Tech Module/Software Technologies/33. Exam Preparation 5/C# Solution/AnimeList/Models/Anime.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace AnimeList.Models
{
    public class Anime
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int Rating { get; set; }


        [Required]
        [StringLength(255, MinimumLength =1, ErrorMessage = "Non-empty text!")]
        public string Name { get; set; }

        [Required]
        [StringLength(255, MinimumLength = 1, ErrorMessage = "Non-empty text!")]
        public string Description { get; set; }

        [Required]
        [StringLength(255, MinimumLength = 1, ErrorMessage = "Non-empty text!")]
        public string Watched { get; set; }
    }
}

File: 02. Tech Module/Software Technologies/33. Exam Preparation 5/JavaScript Solution/models/Anime.js
const mongoose = require('mongoose');

let animeSchema = mongoose.Schema({
    rating: {type: 'number', required: 'true'},
    name: {type: 'string', required: 'true'},
    description: {type: 'string', required: 'true'},
    watched: {type: 'string', required: 'true'},
});

let Anime = mongoose.model('Anime', animeSchema);

module.exports = Anime;